adc/onboard_temperature/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+add_executable(onboard_temperature onboard_temperature.c)
+
+target_link_libraries(onboard_temperature pico_stdlib hardware_adc)
+
+# enable usb output, disable uart output
```

Comment:
I think most of the pico-examples have uart-output-enabled, usb-output-disabled? (because it's easier to debug with uart-output). Might be best to stick to that "convention"? It's easy enough to change back to usb-output in your local copy :wink: